packages/app-file-manager/src/admin/menus/menus.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import { Route } from "@webiny/react-router";
+import { Helmet } from "react-helmet";
+import { AdminLayout } from "@webiny/app-admin/components/AdminLayout";
+import FileManagerSettings from "../views/FileManagerSettings";
+import { SecureRoute } from "@webiny/app-security/components";
+import { RoutePlugin } from "@webiny/app/types";
+import { i18n } from "@webiny/app/i18n";
+import { AdminMenuSettingsPlugin } from '@webiny/app-admin/types';
+
+const t = i18n.ns("app-file-manager/admin");
+
+const ROLE_FM_SETTINGS = ["cms:file-manager:crud"];
```

Comment:
Please rename `cms:file-manager:crud`, to just `files:settings`. I checked the API files, and `files:...` scopes are what we've been using.

BTW I know scopes like `files:crud` are super wide, and should be more specific. It's something we're planning to tackle, hopefully, in the near future.

Reply:
Yeah a lot of this I've been learning by just comparing plugins to understand how the whole project architecture works together, and the scope implementation is something I still haven't fully, deeply investigated. I'll make sure to update this!